src/nu/validator/datatype/Charset.java

```diff
@@ -22,11 +22,6 @@
 
 package nu.validator.datatype;
 
-import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-
```

Comment:
I guess I was wrong about getting rid of ALL the unused imports with the last commit. Whoops! No harm done at least.